### DISCUSSION TITLE
Text View

### DISCUSSION THREAD

**Discussion by pyaephyolinn** [Sketchware-Pro/Sketchware-Pro#932] [Q&A] [answered]
I want use to "LineBreak" in the TextView
Who Help Me ? Please help Me "Code" or "Block set"

**Comment by CyberXCoder-6649** [comment, marked as answer]
I think you should ask help in Any Sketchware community not on github. BTW you can use '\n' for new line (LineBreak) like for example yourTextView.setText("This is text\n and this is new line");
